### DISCUSSION TITLE
How would I configure projectile to tail the output of the compile command?

### DISCUSSION THREAD

**Discussion by johnhamelink** [bbatsov/projectile#1689] [Q&A]
As the title says, really: how would I configure projectile to tail the output of a command such as compile?

I tried using comint mode, and when I do that, I see that one of the commands is  `comint-show-maximum-output`, which when executed does seem to tail the buffer's content as it streams in, but I'm unsure how to trigger this command automatically when the compile window is created.

This is what made sense to my feeble elisp-novice brain 😂 (in my init.el): 
```
(add-hook 'comint-mode-hook (lambda () (comint-show-maximum-output)))
```

However, when I run `projectile-compile-project`,  I get a message in `*messages*` saying, "‘recenter’ing a window that does not display current-buffer."

Now I'm stumped. I'm fairly sure this is trivial if you actually know what you are doing 😇

**Comment by cfclrk** [comment]
When I run `M-x projectile-compile-project`, it pops up a buffer called `*compilation*` (which uses `compilation-mode`) that tails the output of the compilation command as it's running. I guess that's not happening for you?

How do you currently have projectile configured?

**Reply by johnhamelink** [reply]
@cfclrk unfortunately I don’t use projectile anymore - I switched to using project.el.
